common/config/rush/version-policies.json

```diff
@@ -7,7 +7,7 @@
   {
     "policyName": "markup-frontstage",
     "definitionName": "individualVersion",
-    "lockedMajor": 2
+    "lockedMajor": 1
```

Comment:
Can you remove the lockedMajor entirely from this policy?  That should let rush change bump the major version.  Not sure if that's the best way to handle this but would at least let you move forward.

Reply:
removed please! after that I was able to generate new correct changelog with rush change with "major" option.